magento/pwa-studio
# [Fix bug]: [Cart Page] Product Name link is extended till Kebab menu.

## Description

In the CartPage, we made product names and images links that navigate back to the product, but accidentally made the entire grid area clickable. Only the name should be clickable.

## Related Issue





Closes #2752.

## Acceptance 



### Verification Stakeholders

### Specification



### Verification Steps


1. Add a product to cart.
2. Go to cart page and observe the Product name link area.
3. Make sure product name link area is fit with content now.

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
